src/webserver.h

```diff
@@ -1,9 +1,12 @@
 #pragma once
+#include <thread>
 #include <boost/asio.hpp>
 #include "options.h"
 #include "request_handler.h"
 #include "multimap_counter.h"
 
+const int DEFAULT_NUM_THREADS = 8;
```

Comment:
We should change this so that we allow numbers of threads to be determined in the config and have the options struct hold the value of the number of threads. I can implement that